packages/manager/modules/pci/src/projects/project/storages/containers/containers.controller.js

```diff
@@ -59,10 +59,8 @@ export default class PciStoragesContainersController {
       let solution;
       if (!container.s3StorageType) {
         solution = OBJECT_CONTAINER_OFFERS_TYPES.SWIFT;
-      } else if (container.isHighPerfStorage) {
-        solution = OBJECT_CONTAINER_OFFERS_TYPES.HIGH_PERFORMANCE;
       } else {
-        solution = OBJECT_CONTAINER_OFFERS_TYPES.STORAGE_STANDARD;
+        solution = OBJECT_CONTAINER_OFFERS_TYPES.S3;
```

Comment:
I think the addContainerSolution function can be to simplify.

```suggestion
addContainerSolution() {
    this.containers = this.containers.map((container) => ({
        ...container,
        containerSolution: !container.s3StorageType ? OBJECT_CONTAINER_OFFERS_TYPES.SWIFT : OBJECT_CONTAINER_OFFERS_TYPES.S3,
      }));
}
```